core/src/test/kotlin/org/evomaster/core/sql/SqlInsertBuilderTest.kt

```diff
@@ -1071,5 +1071,223 @@ class SqlInsertBuilderTest {
         assertTrue(enabled.subList(1,3).map { it.table.name }.containsAll(listOf("NODEB", "NODEC")))
     }
 
+    ////// Tests Numeric parsing for Min/Max Bounds
```

Comment:
This tests were moved to the corresponding file, they are already in master branch